Calling/MainPage.xaml.cs

```diff
@@ -580,5 +580,36 @@ private void TeamsCaptions_CaptionsReceived(object sender, TeamsCaptionsReceived
         #endregion
 #endif
 #endregion
+
+        public async Task HandlePushNotificationIncomingCallAsync(string notificationContent)
+        {
+            if (this.callAgent != null)
+            {
+                PushNotificationDetails pnDetails = PushNotificationDetails.Parse(notificationContent);

```

Comment:
Please follow the same conversation in this sample to use `var` when declare variables.